app/views/external_pledges/create.js.erb

```diff
@@ -1 +1,2 @@
-// rerender #existing-external-pledges
\ No newline at end of file
+$('#existing-external-pledges').empty().append('<%= escape_javascript(render partial: "external_pledges/external_pledges_table", locals: { patient: @patient, external_pledges: @patient.external_pledges.order_by('created_at desc') } ) %>');
+$('#new-external-pledge').empty().append('<%= escape_javascript(render partial: "external_pledges/new", locals: { patient: @patient, new_external_pledge: @new_external_pledge } ) %>');
```

Comment:
(Ah, I see -- the reason it's getting flushed out is because the `external_pledges_controller`, which doesn't have an instance variable for `@new_external_pledge`, is what's triggering the action. So rails isn't kicking over a new external pledge from the patients controller to the external pledges create controller. Makes sense, awkwardly.)